components/book-demo/Headers.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <div class="container-fluid">
+        <div class="hero container">
+            <div class="row">
+                <div class="col-12 col-md-5 position-relative">
+                    <img class="img-fluid" src="/landing/ee/background.svg" alt="Illustration of Kestra's logo with a dark background" data-aos="zoom-in"/>
+                    <div class="schedule-demo position-absolute">
+                        <div class="subtitle">SCHEDULE A DEMO</div>
+                        <h1>Ready to See Kestra in Action?</h1>
+                        <p class="description">Connect with one of our Product Specialist to get the best out of our product and its value proposition</p>
+                    </div>
+                </div>
+                <div class="col-12 col-md-6 align-items-center d-flex">
+                    <div class="card">
+                        <div class="card-body">
+                            <div class="card-text hs-form-text">
+                            </div>
```

Comment:
I don't know if you have access to the hubspot form but it should be Schedule a Demo and not "submit" for the validation button (it should also be larger)

Reply:
It misses the legend "* Mandatory fields"

Reply:
"*" character for required fields should be purple and prepended with space (see Figma)

Reply:
Book a demo header should be black and not grey

Reply:
Submit button ("Schedule a Demo" after you renamed it) should have the same hover effect as the Get started button on the top bar

Reply:
@brian-mulier-p I don't have the access for the hubspot form, just got the embeded script for the form.

Reply:
I've updated the demo CTA on hubspot. 

For context: first time using an embedded Hubspot form instead of API form. we might see some differences on design side but this will allow to prevent Spam and have more flexibility in the future